src/main/kotlin/org/openredstone/trialore/TrialCommand.kt

```diff
@@ -96,6 +128,12 @@ class TrialCommand(
         if (!app.startsWith("https://discourse.openredstone.org/")) {
             throw TrialOreException("Invalid app: $app")
         }
+        
+        // Warn of recent trials
+        if (!this.isTrialOkay(testificate.uniqueId)) {
```

Comment:
It'd be nice if this gets stated and halts the start of the trial. Could be ignored with an extra argument, like `/trialstart Nickster258 https://app.link confirm` or something.